transport/http/constants.go

```diff
@@ -77,8 +81,38 @@ const (
 
 	// An error occurred. The response body contains an application header.
 	ApplicationErrorStatus = "error"
+
+	// AcceptTrue is the true value for accept headers.
+	AcceptTrue = "true"
```

Comment:
Maybe consider "accept" only, and absence means disallow. Up to you.

Reply:
Yea I think I'm going to move to that model, same as in https://github.com/yarpc/yarpc-go/pull/1318